libs/src/utils/network.js

```diff
@@ -1,7 +1,7 @@
 const axios = require('axios')
 const semver = require('semver')
 
-const Utils = require('../utils')
+const Utils = require('./utils')
```

Comment:
more clear to explicitly `require('./utils.js')`

Reply:
i dont think we should do .js

Reply:
isn't convention to append .js for file imports to clarify dir vs file